src/banner.js

```diff
@@ -1,11 +1,13 @@
+// Using pattern defined here
+// http://ifandelse.com/its-not-hard-making-your-library-support-amd-and-commonjs/
 (function (glob, factory) {
     // AMD support
-    if (typeof define == "function" && define.amd) {
+    if (typeof define === "function" && define.amd) {
```

Comment:
Double-equals introduce subtle errors due to typecasting that may occur. Triple equals skips the type conversions, and is a bit safer. I don't mind removing this line if it bothers you though.